config/logit/20_custom_filters.conf

```diff
@@ -34,6 +34,14 @@ if [@source][component] == "vcap_nginx_access" {
       }
   }
 }
+if [@source][component] == "cloud_controller_ng" {
+  grok {
+    match => {
+      "@message" =>
+      'Started %{WORD:[cloud_controller_ng][method]} "%{URIPATHPARAM:[cloud_controller_ng][uri]}" for user: (%{UUID:[cloud_controller_ng][user]})?, ip: %{IP:[cloud_controller_ng][sourceip]}'
```

Comment:
`source_ip` would be a better name for the last field

Reply:
Addressed